parsec/core/backend_connection/anonymous.py

```diff
@@ -90,21 +95,31 @@ async def pki_enrollment_submit(
     submit_payload_signature: bytes,
     submit_payload: bytes,
 ) -> PkiEnrollmentSubmitRep:
-    rep = cast(
-        PkiEnrollmentSubmitRep,
-        await _anonymous_cmd(
-            serializer=pki_enrollment_submit_serializer,
-            cmd="pki_enrollment_submit",
-            addr=addr,
-            organization_id=addr.organization_id,
+    if FEATURE_FLAGS["UNSTABLE_OXIDIZED_CLIENT_CONNECTION"]:
+        rep = await AnonymousCmds(addr).pki_enrollment_submit(
```

Comment:
Should we re-build every time the HTTP client ?

Reply:
It's the same in python, I understand that is called very rarely so we don't really care, it's like reqwest::get, where we don't keep the client